documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,255 @@ You may obtain a copy of the License at
+specialized Event (_Checkpoint Event_) to signal each Reader to preserve its
+state. Once a Checkpoint has been completed, the application can use the
+Checkpoint to reset all the Readers in the Reader Group to the known consistent
 state represented by the Checkpoint.
 
-For more details on working with ReaderGroups, 
-see [ReaderGroup Basics](basic-reader-and-writer.md#readergroup-basics).
+For more details on working with Reader Groups, Please see [Reader Group Basics](basic-reader-and-writer.md#readergroup-basics).
 
 
 ## Transactions
 
-Pravega supports Transactions.  The idea of a Transaction is that a Writer can
-"batch" up a bunch of Events and commit them as a unit into a Stream.  This is
-useful, for example, with Flink jobs, using Pravega as a sink.  The Flink job
-can continuously produce results of some data processing and use the Transaction
-to durably accumulate the results of the processing.  At the end of some sort of
-time window (for example) the Flink job can commit the Transaction and therefore
+Pravega supports Transactions. The idea of a transaction is that a Writer can
+"batch" up a bunch of Events and commit them as a unit into a Stream. This is
+useful, for example, in Flink jobs, Pravega is used as a sink.  The Flink job
+can continuously produce results for some data processing and use the transaction
+to durably accumulate the results of the processing. For example, at the end of some sort of
+time window, the Flink job can commit the transaction and therefore
 make the results of the processing available for downstream processing, or in
-the case of an error, abort the Transaction and the results disappear.
-
-A key difference between Pravega's Transactions and similar approaches (such as
-Kafka's producer-side batching) is related to durability.  Events added to a
-Transaction are durable when the Event is ack'd back to the Writer.  However,
-the Events in the Transaction are NOT visible to readers until the Transaction
-is committed by the Writer.  A Transaction is a lot like a Stream; a Transaction
-is associated with multiple Stream Segments.  When an Event is published into a
-Transaction, the Event itself is appended to a Stream Segment of the
-Transaction.  Say a Stream had 5 Segments, when a Transaction is created on that
-Stream, conceptually that Transaction also has 5 Segments.  When an Event is
-published into the Transaction, it is routed to the same numbered Segment as if
-it were published to the Stream itself (if the Event would have been placed in
-Segment 3 in the "real" Stream, then it will appear in Segment 3 of the
-Transaction).  When the Transaction is committed, each of the Transaction's
-Segments is automatically appended to the corresponding Segment in the real
-Stream.  If the Stream is aborted, the Transaction, all its Segments and all the
-Events published into the Transaction are removed from Pravega.
+the case of an error, the transaction is aborted and the results disappear.
```

Comment:
Add this sentence: `Therefore, Pravega offers exactly-once guarantees as a storage sink.`